util/udev/50-qmk.rules

```diff
@@ -63,3 +63,9 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="2a03", ATTRS{idProduct}=="0037", TAG+="uacc
 
 # hid_listen
 KERNEL=="hidraw*", MODE="0660", GROUP="plugdev", TAG+="uaccess", TAG+="udev-acl"
+
+# hid bootloaders
+## QMK HID
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="2076", TAG+="uaccess"
```

Comment:
```suggestion
SUBSYSTEMS=="usb", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="2067", TAG+="uaccess"
```

Reply:
Verified the file with this fix — works for flashing an ATmega32U4 with nanoBoot (which uses the same USB IDs and flashing tools as the LUFA/QMK HID bootloader).

Reply:
We've just started the process of migrating our udev rules to a dedicated repo and tool, could we make sure the same changes are applied there? 

https://github.com/qmk/qmk_udev

Reply:
@skullydazed that was the plan. :)
But only so much I can do at 1-2AM. :D